frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardParentToMatchBranchAction.java

```diff
@@ -122,10 +121,11 @@ private void doFastForward(Project project,
       IGitMacheteNonRootBranch gitMacheteNonRootBranch) {
     var localFullName = gitMacheteNonRootBranch.getFullName();
     var parentLocalFullName = gitMacheteNonRootBranch.getParentBranch().getFullName();
-    var refspecChildParent = "${localFullName}:${parentLocalFullName}";
+    // There is no leading '+' in the refspec since we only ever want a fast-forward update.
+    var refspecFromChildToParent = "${localFullName}:${parentLocalFullName}";
 
     // Remote set to '.' (dot) is just the local repository.
-    new FetchBackgroundable(project, gitRepository, refspecChildParent, GitRemote.DOT,
+    new FetchBackgroundable(project, gitRepository, ".", refspecFromChildToParent,
```

Comment:
> Why not GitRemote.DOT?

So that we don't need to get a git4idea's `GitRemote` instance in the code that calls creates `FetchBackgroundable`

> Maybe expose our constant from FetchBackgroundable?

Done